packages/back-end/src/models/ProjectModel.ts

```diff
@@ -122,4 +118,69 @@ export class ProjectModel extends BaseClass {
       },
     };
   }
+
+  protected async afterCreate(
+    doc: ProjectInterface,
+    writeOptions?: never,
+  ): Promise<void> {
+    await super.afterCreate(doc, writeOptions);
+    // Refresh SDK payload cache for all environments
+    const payloadKeys = getPayloadKeysForAllEnvs(
+      this.context as ReqContext | ApiReqContext,
+      [doc.id],
+    );
+    refreshSDKPayloadCache(
+      this.context as ReqContext | ApiReqContext,
+      payloadKeys,
+    ).catch((e) => {
+      logger.error(
+        e,
+        "Error refreshing SDK payload cache after project create",
+      );
+    });
+  }
+
+  protected async afterUpdate(
+    existing: ProjectInterface,
+    updates: Partial<ProjectInterface>,
+    newDoc: ProjectInterface,
+    writeOptions?: never,
+  ): Promise<void> {
+    await super.afterUpdate(existing, updates, newDoc, writeOptions);
+    // Refresh SDK payload cache for all environments
+    const payloadKeys = getPayloadKeysForAllEnvs(
+      this.context as ReqContext | ApiReqContext,
+      [newDoc.id],
+    );
+    refreshSDKPayloadCache(
+      this.context as ReqContext | ApiReqContext,
+      payloadKeys,
+    ).catch((e) => {
+      logger.error(
+        e,
+        "Error refreshing SDK payload cache after project update",
+      );
+    });
+  }
+
+  protected async afterDelete(
+    doc: ProjectInterface,
+    writeOptions?: never,
+  ): Promise<void> {
```

Comment:
I'm also not sure if we really need this afterDelete hook, but I'm fine leaving it in.  Typically, when a project is deleted, features inside the project would also be deleted, which would already trigger any relevant payload refreshes.